setup.cfg

```diff
@@ -1,2 +0,0 @@
-[nosetests]
-exclude-dir = system_tests
```

Comment:
No, it's because I pushed the branch to https://github.com/GoogleCloudPlatform/gcloud-python as well (see my [comment][1])

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/pull/2241#issuecomment-244459343